### PR TITLE
Remove Close Button

### PR DESCRIPTION
**Pull request by DosKit**
Removed the Close Button as well as the option to escape the menu only when you are creating a new character skin to help prevent writing to database issue.

### CODE REVIEW

**Review by IDKFORCE — APPROVED** [review]
this looks good good job

**Comment by gononono64** [comment]
Got this error creating new character MainThrd/ error during NUI callback createNewCharacter: @qbr-multicharacter/client/main.lua:65: No such export RequestAndSetModel in resource qbr-clothing

File: fxmanifest.lua [hunk 1]
```diff
@@ -35,3 +35,5 @@ files {
 dependencies {
 	'qbr-core'
 }
+
+lua54 'yes
```
Comment (gononono64) [inline fxmanifest.lua:39]: Missing a '
`lua54 'yes`
should be
`lua54 'yes'`

Comment (DosKit) [reply fxmanifest.lua]: This Been Corrected

**Pull request merged**